robotbase/src/main/java/frc/robot/commands/ChoreoTrajectoryCommand.java

```diff
@@ -20,24 +21,39 @@ public class ChoreoTrajectoryCommand extends SequentialCommandGroup {
    * @param trajectoryFileName The name of the path file with '.traj' excluded.
    */
   public ChoreoTrajectoryCommand(String trajectoryFileName) {
-    this(trajectoryFileName, trajectoryFileName);
+    this(trajectoryFileName, trajectoryFileName, true);
+  }
+
+  /**
+   * @param trajectoryFileName The name of the path file with '.traj' excluded.
+   * @param pathName The name of the path, is returned in the toString for the auto command chooser.
+   */
+  public ChoreoTrajectoryCommand(String trajectoryFileName, String pathName) {
+    this(trajectoryFileName, pathName, true);
   }
 
   /**
    * A utility command to run a Choreo path correctly.
    *
    * @param trajectoryFileName The name of the path file with '.traj' excluded.
    * @param pathName The name of the path, is returned in the toString for the auto command chooser.
+   * @param setPoseToStartTrajectory Whether or not to set the robot pose to the paths starting
+   *     trajectory.
    */
-  public ChoreoTrajectoryCommand(String trajectoryFileName, String pathName) {
+  public ChoreoTrajectoryCommand(
+      String trajectoryFileName, String pathName, boolean setPoseToStartTrajectory) {
     this.trajectoryFileName = trajectoryFileName;
```

Comment:
Remember that we have decided to use the m_ prefix to denote member variables instead of using the `this` keyword.